modules/steam/automount.nix

```diff
@@ -0,0 +1,83 @@
+{lib, config, pkgs, ...}: let
+  inherit (lib) mkIf mkMerge mkOption types mdDoc mkEnableOption;
+  inherit (lib.strings) concatMapStringsSep;
+  inherit (pkgs) systemd util-linux procps steam writeShellScriptBin ;
+  cfg = config.jovian.steam;
+  mount-script = writeShellScriptBin "mount-script" ''
+#Steam Deck Mount External Drive by scawp
+#License: DBAD: https://github.com/scawp/Steam-Deck.Mount-External-Drive/blob/main/LICENSE.md
+#Source: https://github.com/scawp/Steam-Deck.Mount-External-Drive
```

Comment:
This license text is significantly problematic enough that I don't want to have anything "licensed" under that license within the project.

It's similar to those other licenses where you have to give a moral judgement on an action, which means that if it is even enforceable, it is vague enough to mean anything.

(This comment is just about the license, this is a desirable feature...)